### ISSUE TITLE
Unexpected `lsp-find-definition` behavior

### ISSUE THREAD

**Issue by akircanski** [erlang-ls/erlang_ls#1400]
For a newcomer, expected behavior for `lsp-find-definition` would a jump to a function definition, even if the function is defined in another module. Example:

```erlang
-module(geometry).
-export([area/1]).

area({rectangle, Width, Ht}) -> Width * Ht;
area({circle, R})            -> 3.14159 * R * R.
```

```erlang
-module(usegeometry).
-export([testArea/0]).

-import(geometry,[area/1]).

testArea() -> area({circle,2}).
```

Running `lsp-find-definition` against `area` in the second module (in the `testArea() -> area({circle,2})`)  does not jump to the first module's definition of area. It jumps to (and highlights) that same line in the `usegeometry` module.

Is this expected behavior and why? What Erlang LS function should be used for jumping to the actual definition?

Environment: Emacs 28.2, started with `emacs -l ~/.emacs.d/erlang-lsp-init.el usegeometry.erl`, where `erlang-lsp-init.el` is the config setup file from the official Erlang LS. Erlang/OTP 25.

**Comment by plux** [comment]
This should work. 

Are you sure that erlang_ls is running? What does *lsp-log* say?

Make sure you put the `.erl` files inside `src/` directory if you haven't done that.

**Comment by akircanski** [comment]
Yes, you are right, thank you! The issue was: when I added the project (opened it in Emacs with Erlang LS mode for the first time), I imported a directory that did not contain the `src/` subdirectory. In my case, `src/ does not live at the top level of the project.  

It may make sense to modify that package import prompt to mention that it's necessary to import a directory containing `src/` - there are likely projects with diverse directory structures (around/unrelated to Erlang code).
